acceptance-tests/dsl/src/main/java/org/hyperledger/besu/tests/acceptance/dsl/privacy/condition/ExpectValidPrivateTransactionReceipt.java

```diff
@@ -42,7 +42,16 @@ public void verify(final PrivacyNode node) {
         node.execute(transactions.getPrivateTransactionReceipt(transactionHash));
     assertThat(actualReceipt)
         .usingRecursiveComparison()
-        .ignoringFields("commitmentHash", "logs")
+        .ignoringFields(
+            "commitmentHash",
+            "logs",
+            "blockHash",
+            "blockNumber",
+            "logsBloom",
+            "transactionIndex") // TODO: The fields blockHash, blockNumber, logsBloom and
```

Comment:
NIT: This shouldn't be a TODO. It is just a note isn't it?

Reply:
This really is a problem with the PrivateTransactionReceipt defined in web3j. Once this is updated these can be removed. That is why I put it in as a TODO ...